spec/features/movies_feature_spec.rb

```diff
@@ -80,7 +86,7 @@
         VCR.use_cassette('update_movie') do
           click_link("update_movie_link_movie_show")
         end
-        expect(page).to have_content("98 min")
+        expect(page).to have_content("1h 38m")
```

Comment:
Unrelated. Just noticed it was failing, and we apparently never updated it when we changed the minute count display.